.yarn/versions/23967d0d.yml

```
@@ -0,0 +1,33 @@
+releases:
```

You don't need to mark all packages for release - just mark both the fslib and the CLI as patch releases, and set everything else on "decline".

---

Well `@yarnpkg/plugin-pnp` and `@yarnpkg/pnp` should be released as well since the PnP hook needs to be rebuilt

---

@arcanis @merceyz, I made the requested changes in the code. Note the sync version of `fs.exists` had already a catch all exception attached, so I left that function as is.